backend/models.py

```diff
@@ -0,0 +1,77 @@
+from pydantic import BaseModel
+
+def make_deck(x):
+    '''
+    Makes a deck composing of x of each card
+
+    Parameters:
+        x (int): The amount of each card to add to deck
+    '''
+
+    # Single character representation of a deck of cards, no suits
+    cards = ["A","2","3","4","5","6","7","8","9","T","J","Q","K"]
+
+    # Add x of each card to deck
+    deck = []
+    
+    for card in cards:
+        for i in range(x):
+            deck.append(card)
+
+    return deck
+
+class Lobby(BaseModel):
+    '''
+    A class representing a game state
+
+    Attributes:
+        id (int): The lobby's internal ID
+        code (str): The code used to join the lobby
+        players (dict): A list of player objects participating in the game
+        player_count (int): Number of players in game, max 4
+
+        started (bool): Whether the game has started
+
+        current_turn (int): The ID of the player whose turn it current is
+
+    Methods:
+        get_deck_count(): Returns size of the deck
```

Comment:
This is minor but I would prefer this to be in a docstring on the method, rather in the class docstring. For future reference.